src/modules/[chain]/uptime/index.vue

```diff
@@ -30,7 +34,7 @@ onMounted(() => {
       // constructs sequence for loading blocks
       let promise = Promise.resolve()
       for (let i = height - 1; i > height - 50; i -= 1) {
-        if (i > height - 48 && i > 0) {
+        if (i > height - 50 && i > 0) {
```

Comment:
No need be 50, since we will periodically load latest blocks. it's last few blocks will possibly be useless.